src/main/java/frc/robot/RobotContainer.java

```diff
@@ -103,26 +148,103 @@ public void configureDefaultCommands() {
    * {@link JoystickButton}.
    */
     private void configureButtonBindings() {
```

Comment:
Suggest making a separate class just for controller IO

Reply:
Also consider separating the controls per controller ie. Driver, Operator, and Button Board

Reply:
controls are seperate no?

Reply:
Is Driver and Operator separated? The method names don't really relate to that.

Reply:
we can talk abt this when ur in next time

Reply:
Discussed with Tim, moving each controllers to their own method.